### PR TITLE
removing doubt in the sentence

### PR DESCRIPTION
**Pull request by bayethiernodiop**
this may lead to think that in the integer encoding different words can have similar number to represent them.

### CODE REVIEW

**Comment by adammichaelwood** [comment]
what about:

"There is no relationship between the similarity of any two words and the similarity of their encodings."

**Comment by bayethiernodiop** [comment]
@adammichaelwood agree with you

File: site/en/r2/tutorials/sequences/word_embeddings.ipynb [hunk 1]
```diff
@@ -103,7 +103,7 @@
         "\n",
         "* The integer-encoding is arbitrary (it does not capture any relationship between words).\n",
         "\n",
-        "* An integer-encoding can be challenging for a model to interpret. A linear classifier, for example, learns a single weight for each feature. Because different words may have a similar encoding, this feature-weight combination is not meaningful.\n",
+        "* An integer-encoding can be challenging for a model to interpret. A linear classifier, for example, learns a single weight for each feature. Because There is no relationship between the similarity of any two words and the similarity of their encodings, this feature-weight combination is not meaningful.\n",
```
Comment (yashk2810) [inline site/en/r2/tutorials/sequences/word_embeddings.ipynb]: s/There/there

Comment (bayethiernodiop) [reply site/en/r2/tutorials/sequences/word_embeddings.ipynb]: done

**Pull request merged**